packages/postgres-database/src/simcore_postgres_database/models/groups_extra_properties.py

```diff
@@ -42,6 +42,13 @@
         "If a user is part of this group, it's "
         "service can access the internet.",
     ),
+    sa.Column(
+        "override_services_specifications",
```

Comment:
MINOR: suggest to call boolean flag as `can_ override_services_specifications` or `override_services_specs_allowed`

Reply:
so the table *groups_extra_properties* already is containing an entry: internet_access which is a boolean. I would not now start having can_ in here since this is implicitely defined as such. It is a table containing options, so *can_* is implicit. But we can discuss that further later.